stylesheets/base.css

```diff
@@ -55,3 +55,7 @@
 .relative {
     position: relative;
 }
+
+.color-red {
+    color: red;
```

Comment:
try to use color variables, if possible!!
the rest of the code looks good!!
you can merge it